src/animation/fluidResistance.ts

```diff
@@ -41,9 +47,45 @@ const applyFluidResistanceForceForStep = (
   });
 
   const gravitationalForce: Vector<number> = [0, mover.mass * gE];
-  const netForce = addf({ v1: dragForce, v2: gravitationalForce });
+  const netForce = addf({
+    v1: dragForce,
+    v2:
+      playState === PlayState.Forward
+        ? gravitationalForce
+        : multf({ v: gravitationalForce, s: -1 }),
```

Comment:
If running the animation in the reverse direction, we need to reverse the direction of the gravitational force. Drag will always operate in the reverse direction of velocity, so that's already taken care of ✅